packages.md

```diff
@@ -35,6 +35,26 @@ Or you can do it with [`LOOP`][loop].
   collect s)
 ~~~
 
+# Give package local nickname
+
+Sometimes it is handy to give imported a local name for saving some typing. 
+
+This can be achieved by using [`RENAME-PACKAGE`][rename-package]. For example:
+
+~~~lisp
+(asdf:load-system :cl-ppcre)
+
+(defpackage :mypackage
+  (:use :cl))
+(in-package :mypackage)
+
+(rename-package :cl-ppcre :re)
```

Comment:
Also a note about `:nickname` in package definitions ?

Reply:
maybe my remark is a better fit for systems.md. (do as you like)

Reply:
I think might be better to clearify that "nickname" mentioned here is different with the one in `DEFPACKAGE`.

I will add it.

Reply:
Fixed. Please check.